backend/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+services:
+  money-tree-application:
+    build: .
+    restart: on-failure
+    ports:
+      - 8080:8080
+    links:
+      - neo4j
+    depends_on:
+      - neo4j
+  neo4j:
+    image: neo4j
+    restart: unless-stopped
```

Comment:
So this line means that the database should be on to always persist data and if it is stopped, we restart it right?

Reply:
It means if the service stops for any reason (docker daemon restarts, vm power cut, ...), it will restart the database service, unless we stopped it ourselves. In our case, this line is optional, and I don't see when it use it, but I put it there for extra robustness in case i missed something.
See documentation [here](https://docs.docker.com/config/containers/start-containers-automatically/#use-a-restart-policy)

Reply:
Ok I see! Good stuff.